media-video/webcamoid/webcamoid-8.7.1.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PLOCALES="ca de el es et fr gl he it ja kab ko nb_NO nl pl pt pt_BR ru tr uk zh_CN zh_TW"
+inherit l10n qmake-utils
```

Comment:
Just install all locales unconditionally, helps you to clean this ebuild a bit too.

Reply:
Done.